README.md

```diff
@@ -39,28 +39,95 @@ Prerelease version
 $ npm i iam-client-lib@canary
 ```
 
-### Sample Import (TypeScript)
+### Sample Config for browsers (TypeScript)
+
+```js
+import {
+  IAM,
+  WalletProvider,
+  setCacheClientOptions,
+  setChainConfig,
+  setMessagingOptions,
+  MessagingMethod,
+} from 'iam-client-lib'
+
+export class App {
+    private _iam: IAM;
 
-``` sh
-import { IAM, CacheServerClient } from 'iam-client-lib';
+    constructor() {
+      // IAM will be created with default settings for VOLTA CHAIN
```

Comment:
I don't think is quite the case. I think with Dmitry's changes, we are using whatever chainID we get from the provider: https://github.com/energywebfoundation/iam-client-lib/blob/faeec472c6d9263b984ab85063dc3c66bc9ad0ff/src/iam/iam-base.ts#L620